src/cursor/cur_std.c

```diff
@@ -582,6 +595,100 @@ err:	API_END(session, ret);
 }
 
 /*
+ * __cursor_modify --
+ *	WT_CURSOR->modify default implementation.
+ */
+static int
+__cursor_modify(WT_CURSOR *cursor, WT_MODIFY *entries, int nentries)
+{
+	WT_DECL_RET;
+	WT_SESSION_IMPL *session;
+	WT_DECL_ITEM(ta);
+	WT_DECL_ITEM(tb);
+	WT_DECL_ITEM(tmp);
+	size_t len, size;
+	int i;
+
+	CURSOR_UPDATE_API_CALL(cursor, session, modify, NULL);
+	WT_ERR(__cursor_checkkey(cursor));
+
+	/* Check for a rational damage vector count. */
```

Comment:
I don't love the ``damage vector`` naming, I'd prefer to keep it out of the WiredTiger code.